tests/kernel/usage/thread_runtime_stats/src/test_thread_runtime_stats.c

```diff
@@ -212,7 +212,7 @@ ZTEST(usage_api, test_thread_stats_enable_disable)
 	k_thread_runtime_stats_get(arch_current_thread(), &stats1);
 	k_thread_runtime_stats_get(tid, &helper_stats1);
 	k_thread_runtime_stats_disable(tid);
-
+	zassert_false(k_thread_runtime_stats_is_enabled(tid));
```

Comment:
Should have kept the empty line
```suggestion
	zassert_false(k_thread_runtime_stats_is_enabled(tid));

```